packages/@aws-cdk/core/test/duration.test.ts

```diff
@@ -4,8 +4,13 @@ import { Duration, Lazy, Stack, Token } from '../lib';
 describe('duration', () => {
   test('negative amount', () => {
     expect(() => Duration.seconds(-1)).toThrow(/negative/);
+  });
 
+  test('can stringify but not resolve', () => {
+    const stringified = `${Duration.hours(1)}`;
```

Comment:
Perhaps it makes more sense to simply return a string representation of the duration (eg `"1 hour"` or even just `״<duration>"`).

Reply:
You are saying get rid of the misuse protection because it's not worth it, right?

Reply:
Yap

Reply:
ok